VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitFragment.java

```diff
@@ -14,6 +14,8 @@
 public abstract class CourseUnitFragment extends BaseFragment implements PageViewStateCallback, RunnableCourseComponent {
     public interface HasComponent {
         CourseComponent getComponent();
+        boolean navigateNextComponent();
```

Comment:
Removed.

On Wed, Aug 3, 2016 at 2:17 PM, Christopher Lee <notifications@github.com>
wrote:

> In VideoLocker/src/main/java/org/edx/mobile/view/CourseUnitFragment.java
> <https://github.com/edx/edx-app-android/pull/761#discussion_r73389486>:
>
> > @@ -14,6 +14,8 @@
> >  public abstract class CourseUnitFragment extends BaseFragment implements PageViewStateCallback, RunnableCourseComponent {
> >      public interface HasComponent {
> >          CourseComponent getComponent();
> > +        boolean navigateNextComponent();
>
> If nothing uses it now and there is no concrete plan or reason to use it
> in the future, than remove it.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/edx/edx-app-android/pull/761/files/64c1f08c0f3651fb25188c643146d7accc62968d#r73389486>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AR10TxmN0NIDfbzKyC5fR3X-qPDP-EsKks5qcNtKgaJpZM4JCZsX>
> .
>



-- 
Regards,

Marc DiNino
Android Engineer
EDX